pkg/parser/model/model.go

```diff
@@ -1319,6 +1319,14 @@ func (pi *PartitionInfo) HasTruncatingPartitionID(pid int64) bool {
 	return false
 }
 
+// ClearReorgIntermediateInfo remove intermediate information used during reorganize partition.
+func (pi *PartitionInfo) ClearReorgIntermediateInfo() {
```

Comment:
Please add a simple UT to model_test.go.

Reply:
Test is added. https://github.com/pingcap/tidb/pull/51631/commits/34b8c3a172060e5c2b0b23feb45944b9237f1156